B827EBFFFEB066EC.json

```diff
@@ -0,0 +1,27 @@
+{
+	"gateway_conf": {
+		"gateway_ID": "B827EBFFFEB066EC",
+		 /* Devices*/
+                "gps":true,
+		/* node server for poly packet server*/ 
+		"servers": [
+			{
+				"server_address": "router.as1.thethings.network",
+				"serv_port_up": 1700,
+				"serv_port_down": 1700, 
+				"serv_enabled": true
+			}
+		],
+	         /* GPS configuration */
+                "gps_tty_path":"/dev/ttyS0",
+                "fake_gps":true 
+		"ref_latitude": 1.34535650,
+		"ref_longitude": 103.68069970,
+		"ref_altitude": 58,
+	        /*Contact and Description*/

```

Comment:
Ideally, best is to remove the comments, since they are not really part of json spec.

Reply:
removed comments